src/Elastic.Apm.NLog/readme.md

```diff
@@ -0,0 +1,34 @@
+# Elastic APM NLog Layout Renderer
+
+Allows you to add the following place holders in your NLog templates
+
+* `ElasticApmTraceId`
+* `ElasticApmTransactionId`
+
+Which will be replaced with the appropriate Elastic APM variables if available
+
+
+## How to Enable
+
+```csharp
+			var target = new MemoryTarget();
+			target.Layout = "${ElasticApmTraceId}|${ElasticApmTransactionId}|${message}";
+			Agent.Tracer.CaptureTransaction("TestTransaction", "Test", t =>
+			{
+				traceId = "trace-id";
+				transactionId = "transaction-id";
+				logger.Debug("InTransaction");
+			});
+			// Logged message will be in format of `trace-id|transation-id|InTransaction`
+			// or `||InTransaction` if the place holders are not available
+```
+
+## Prerequisite
+
+The prerequisite for this to work is a configured [Elastic APM Agent](https://github.com/elastic/apm-agent-dotnet). If the agent is not configured the APM place holders will be empty.
+
+## Copyright and License
+
+This software is Copyright (c) 2014-2019 by Elasticsearch BV.
```

Comment:
`2014-2020` ? :)  (I also see this in the root `README.md`)

Reply:
Doh!